client-web/src/functional-e2e/test-plan-applications.md

```diff
@@ -0,0 +1,897 @@
+- All three levels show "Apply" or "Join" button available
+- Non-member cannot directly access space content
+
+#### 1.2 Verify Space Hierarchy Structure
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Navigate to spaces listing
+3. Identify parent-child relationships in the tree
+
+**Expected Results:**
+
+- Space (Level 0) is the root
+- Subspace (Level 1) appears nested under Space
+- Subsubspace (Level 2) appears nested under Subspace
+- Tree structure clearly indicates hierarchy
+
+---
+
+### 2. Application Submission Process
+
+**Seed:** `./client-web/src/functional-e2e/seed-applications.spec.ts`
+
+#### 2.1 Apply to Level 0 Space
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Navigate to the Level 0 Space
+3. Click the "Apply" button
+4. Verify questionnaire modal/form appears
+5. Fill in questionnaire with test answers:
+   - Question 1: "I am interested in collaborating on this space"
+   - Question 2: "5 years of experience in the field"
+6. Submit the application
+
+**Expected Results:**
+
+- Apply button is visible and clickable on Level 0 space
+- Questionnaire modal opens with application questions
+- All form fields are editable
+- Submit button becomes enabled after completing required fields
+- Success message appears after submission
+- Application status changes from "Apply" to "Application Pending" or similar
+
+#### 2.2 Apply to Level 1 Subspace
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Navigate to the Level 1 Subspace
+3. Click the "Apply" button
+4. Fill in questionnaire with test answers:
+   - Question 1: "I want to contribute to this specific subspace"
+   - Question 2: "Expert in relevant technologies"
+5. Submit the application
+
+**Expected Results:**
+
+- Apply button is visible on Level 1 subspace
+- Questionnaire appears specific to the subspace
+- Application is successfully submitted
+- Confirmation message is displayed
+
+#### 2.3 Apply to Level 2 Subsubspace
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Navigate to the Level 2 Subsubspace
+3. Click the "Apply" button
+4. Fill in questionnaire with test answers:
+   - Question 1: "I have specific skills for this subsubspace"
+   - Question 2: "Available 10 hours per week"
+5. Submit the application
+
+**Expected Results:**
+
+- Apply button is visible on Level 2 subsubspace
+- Questionnaire opens correctly
+- Application is successfully submitted
+- User receives confirmation
+
+#### 2.4 Prevent Duplicate Applications
+
+**Steps:**
+
+1. Sign in as `${TestUser.NON_SPACE_MEMBER}@alkem.io`
+2. Apply to Level 0 Space (if not already done)
+3. Attempt to apply again to the same space
+
+**Expected Results:**
+
+- Apply button is disabled or replaced with "Application Pending"
+- System prevents duplicate application submission
+- Clear status indicator shows existing application
```

Comment:
Beside "Application pending" message, we don't display other indicators, so this step could be removed